src/components/KCopy/KCopy.vue

```diff
@@ -0,0 +1,274 @@
+<template>
+  <div
+    v-if="!isBadge"
+    class="k-copy"
+    :class="truncate ? null : 'copy-truncated'"
+  >
+    <KTooltip
+      v-if="format !== 'hidden'"
+      :class="truncate ? null : 'copy-content'"
+      data-testid="copy-content"
+      :label="contentTooltip"
+      placement="bottomStart"
+      position-fixed
+    >
+      <div
+        :class="[
+          'content-container',
+          truncate ? 'truncated-content' : 'non-truncated-content',
+          monospace ? 'monospace' : null
+        ]"
+        :style="widthStyle"
```

Comment:
suggestion: instead of inlining the styles, can you add another dynamic class and control the width in the `<style>` block?

Reply:
widthStyle is calculated based on props value; what do you mean by control it in <style> block ?